docs/resources/group.md

```diff
@@ -23,6 +24,7 @@ The following arguments are supported:
 
 * `name`                - (Required) Name of the group
 * `description`         - (Optional) A description for the group
+* `external_id`         - (Optional) New external group ID used to configure the corresponding group in Azure AD.
```

Comment:
Why the reference to AD azure?

Reply:
That's copied from Artifactory's tooltip (as per our convention).

Reply:
<img width="522" alt="Screen Shot 2022-04-29 at 9 53 02 AM" src="https://user-images.githubusercontent.com/27263/165989067-cc7dd332-30c9-4072-9b28-0f8b112dd839.png">

Reply:
I shut my mouth